### ISSUE TITLE
Issue with "URL()": No mention of creating URLs from anchors?

### ISSUE THREAD

**Issue by kaycebasques** [mdn/content#2767]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/URL/URL

#### What information was incorrect, unhelpful, or incomplete?

I just stumbled upon this. I'm not sure if I'm understanding this correctly. But it seems like you can create URL objects from anchor elements. This doesn't appear to be mentioned in the URL constructor reference.

#### Specific section or headline?

https://developer.mozilla.org/en-US/docs/Web/API/URL/URL#parameters

#### What did you expect to see?

Mention of passing an anchor element as the first argument.

#### Did you test this? If so, how?

https://twitter.com/kaycebasques/status/1366253083127291906?s=20

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/api/url/url`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/URL/URL
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/url/url/index.html
* Last commit: https://github.com/mdn/content/commit/11a98a59fca5f2e448d9e4b70e3c8a538c47b201
* Document last modified: 2021-02-19T19:45:18.000Z

</details>

**Comment by sideshowbarker** [comment]
Thanks for raising.

The behavior described in the issue description is due to fact that anchor element objects have a special stringifier — such that if you pass them as an object to a method that expects a string, the value of the `href` attribute is used.

https://html.spec.whatwg.org/multipage/links.html#api-for-a-and-area-elements

I’ll raise a PR with an update to the https://developer.mozilla.org/en-US/docs/Web/API/URL/URL to state that.

**Comment by whizkydee** [comment]
Yeah, I find that interesting. Contrary to most interfaces that extend `HTMLElement`, `HTMLAnchorElement#toString()` seems to return the `href` of the anchor element in question. Pretty neat.

**Comment by sideshowbarker** [comment]
OK, raised https://github.com/mdn/content/pull/2768 for this
